Shed requests (#1619)
scylladb/scylla-monitoring: docs/source/use-monitoring/advisor/systemOverload.rst

System Overload
---------------

There could be multiple indications that a system is overloaded:

* Timeouts

> * Requests shed - Requests are shed when the system cannot process the requests fast enough.

Reviewer: ```suggestion
* Requests shed - Requests are shed (dropped) when the system cannot process requests fast enough.
```

> * CPU at 100% when no background process (like compaction or repair) is running.

Reviewer: ```suggestion
* CPU at 100% when no background process (like compaction or repair) runs.
```

* Ques are getting filled.

> If you ruled out data-model problems and hardware failure, this could be an indication that you need to rescale the system.

Reviewer: ```suggestion
If you ruled out data-model problems and hardware failure, this could indicate you need to scale the system.
```
